packages/react/src/patterns/sub-patterns/Card/__stories__/Card.stories.js

```diff
@@ -1,39 +1,48 @@
 import './index.scss';
-import { select, text, withKnobs } from '@storybook/addon-knobs';
+import {
+  select,
+  text,
+  withKnobs,
+  boolean,
+  object,
+} from '@storybook/addon-knobs';
 import { ArrowRight20 } from '@carbon/icons-react';
 import { Card } from '../';
 import React from 'react';
 import readme from '../README.md';
 import { storiesOf } from '@storybook/react';
 
+const props = {
+  cardProps: () => ({
+    heading: text('title (required)', 'Lorem ipsum dolor sit amet'),
+    type: text('type', 'link'),
```

Comment:
Would this be better if it was a `select`?

Reply:
Either that or based on how you wrote out this story I wouldn't include it at all (as a knob).

Reply:
@photodow Will make note to address for that component.

Reply:
I think this might still be relevant because when it was moved up here in this pull request we left type in each individual story. So it's overwriting what's here.

Reply:
it was removed